.github/workflows/server-tests.yaml

```diff
@@ -1,10 +1,8 @@
 on:
   pull_request:
-    branches:
-      - main
-      - develop
+    types: [opened, reopened, edited]
     paths:
-      - 'server/**'
+      - "server/**"
```

Comment:
Side Quest: As discussed on omSupply Devs, we want server tests to run on every PR. Added opened, reopened, and edited types to this action. This GH action runs whenever a PR affecting the server directory is opened, reopened, or edited.